packages/Molecules/Form/formPikaday.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+  <input
+    type="text"
+    v-model="displayVal"
+    :autocomplete="autocomplete"
+    :disabled="disabled"
+    :placeholder="placeholder"
+    :readonly="readonly"
+    :name="inputName" />
+</template>
+
+<script>
+import { defaults } from 'lodash'
+import moment from 'moment'
+import Pikaday from 'pikaday'
```

Comment:
While a pikaday widget does exist in the vue-form-generator library, it requires that pikaday is loaded from the head of the web page. If you check this line you can see how it expects window to have a pikaday variable loaded. This seems to go against how ripple is set up, as pikaday should be loaded into the scope of the form component. Because of this I have avoided using the existing element and created a custom wrapper formPikaday.vue.